### PR TITLE
Update demo dependencies and readme files

### PR DESCRIPTION
**Pull request by followyouhome**
Little cleanup of some files

### Check lists

- [x] Readme file updated with actual description and example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+</p>
+
+# Rebilly Recomponents
+
+As the number of Web Apps developed at Rebilly grows larger, it has become apparent that we need a way to share UI components across projects. Recomponents offers similar levels of customization to projects like Twitter Bootstrap, but comes default with the familiar look and feel. This library also delivers consistency of implementation and usage to Rebilly developers (and in the future, the greater Vue community).
+
+## Repository structure
+
+* `packages/recomponents` Recomponents project
+* `packages/grisome` Demo project with Nuxt
```
Comment (ZengJunyong) [inline README.md]: * `packages/grisome` Demo project with grisome

File: README.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+* `packages/nuxt` Demo project with Gridsome
```
Comment (ZengJunyong) [inline README.md]: * `packages/nuxt` Demo project with nuxt

File: README.md (hunk 1 above)
Comment (followyouhome) [reply README.md]: updated

File: README.md (hunk 2 above)
Comment (followyouhome) [reply README.md]: updated

**Pull request merged**